### PR TITLE
use gradle 7.4.1

### PR DESCRIPTION
**Pull request by Jimexist**
Motivation:

minor version upgrade - should be compatible with 7.3.3

Modifications:

- update gradle wrapper version

### CODE REVIEW

**Review by jrhee17 — APPROVED** [review]
Thanks @Jimexist ! 🙇 👍 🙇

**Review by ikhoon — APPROVED** [review]
Thanks, @Jimexist!

File: gradle/wrapper/gradle-wrapper.properties [hunk 1]
```diff
@@ -1,5 +1,5 @@
 distributionBase=GRADLE_USER_HOME
 distributionPath=wrapper/dists
-distributionUrl=https\://services.gradle.org/distributions/gradle-7.3.3-all.zip
+distributionUrl=https\://services.gradle.org/distributions/gradle-7.4-all.zip
```
Comment (anuraaga) [inline gradle/wrapper/gradle-wrapper.properties:3]:
```suggestion
distributionUrl=https\://services.gradle.org/distributions/gradle-7.4.1-all.zip
```

**Review by anuraaga — APPROVED** [review]

**Review by trustin — APPROVED** [review]
Thanks!

**Pull request merged**